packages/build/package.json

```diff
@@ -73,6 +74,8 @@
     "@netlify/plugins-list": "^6.68.0",
     "@netlify/run-utils": "^5.1.1",
     "@netlify/zip-it-and-ship-it": "9.10.0",
+    "@opentelemetry/api": "^1.4.1",
+    "@opentelemetry/instrumentation-http": "^0.40.0",
```

Comment:
Not sure how the upload step works between buildbot and build, but if it's uploading the files via HTTP from this process, this will get slightly noisy.

Reply:
> The HTTP calls we have within build should be mostly to our API

Sounds good then! I just wanted to point out the potential issue, but if it doesn't make 10s of thousands of HTTP calls it's all good.